testsuite/MDAnalysisTests/coordinates/test_gsd.py

```diff
@@ -0,0 +1,38 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+from __future__ import absolute_import
+
+import pytest
+from numpy.testing import assert_almost_equal
+
+from MDAnalysisTests.datafiles import GSD
+
+import MDAnalysis as mda
+
+
+@pytest.fixture
+def GSD_U():
+    return mda.Universe(GSD)
+
+def test_gsd_positions(GSD_U):
```

Comment:
add some more tests for things like:
 - positions at the next frame
 - the total number of frames in the traj
 - the number of atoms in the file
 - dimensions
 - data['step']

Reply:
I'll try to figure out how to do this based on other examples in the testsuite